query-engine/driver-adapters/src/wasm/adapter_method.rs

```diff
@@ -1,25 +1,19 @@
 use js_sys::{Function as JsFunction, Promise as JsPromise};
-use serde::Serialize;
-use serde_wasm_bindgen::Serializer;
 use std::marker::PhantomData;
 use wasm_bindgen::convert::FromWasmAbi;
 use wasm_bindgen::describe::WasmDescribe;
-use wasm_bindgen::{JsCast, JsError, JsValue};
+use wasm_bindgen::{JsCast, JsValue};
 use wasm_bindgen_futures::JsFuture;
 
 use super::error::into_quaint_error;
 use super::from_js::FromJsValue;
+use super::to_js::ToJsValue;
 use crate::AdapterResult;
 
-// `serialize_missing_as_null` is required to make sure that "empty" values (e.g., `None` and `()`)
-//  are serialized as `null` and not `undefined`.
-// This is due to certain drivers (e.g., LibSQL) not supporting `undefined` values.
-pub(crate) static SERIALIZER: Serializer = Serializer::new().serialize_missing_as_null(true);
```

Comment:
Note from my side: this will have impacts to my PR https://github.com/prisma/prisma-engines/pull/4648